docs/csharp/how-to/index.md

```diff
@@ -0,0 +1,241 @@
+---
+title: How to articles (C# Guide)
+description: A collection of quick tips and short, focused code samples
+author: billwagner
+ms.author: wiwagn
+ms.date: 12/20/2017
+ms.topic: article
+ms.prod: .net
+ms.devlang: devlang-csharp
+---
+
+# How to (C#)
+
+The How to section of the C# Guide is where you can find quick answers
+to common questions. There are cases on this page were one article may
```

Comment:
In some cases, articles are listed in multiple sections on this page. We want...